### PR TITLE
Multicluster app

### PR DESCRIPTION
**Pull request by MBishop17**
@deniseschannon and @alena1108, this PR still needs step-by-step instruction, as the UI wasn't checked in to master when drafting.

### CODE REVIEW

File: content/rancher/v2.x/en/k8s-in-rancher/workloads/_index.md [hunk 1]
```diff
@@ -41,7 +61,40 @@ Kubernetes divides workloads into different types. The most popular types suppor
...
+
+_Available as of v2.2.0_
+
+In use cases where you need to deploy a common workload to multiple clusters or projects, deploy it using a multi-cluster application. A multi-cluster application is a Helm chart template that stores a unique workload configuration. When you deploy a multi-cluster application, this template replicates a unique workload for each deployment target (i.e., clusters or projects) that you choose.
+
+Multi-cluster applications are beneficial because they reduce the repetition of deploying individual workloads to each target. They also reduce the likelihood of user error during workload configuration, as you only have to configure a single object rather than a unique object for each target. Additionally, because multi-cluster workloads are a single object, they simplify routine maintenance. For example, rather than upgrading several unique workloads, you can upgrade the multi-cluster application, which then replicates to each target. 
+
+You can deploy a multi-cluster application using either a regular deployment or a catalog deployment. Multi-cluster workloads deployed by Helm chart can be upgraded or reconfigured by editing its catalog application object.
```
Comment (alena1108) [inline content/rancher/v2.x/en/k8s-in-rancher/workloads/_index.md]: what is `a regular deployment`?

File: content/rancher/v2.x/en/k8s-in-rancher/workloads/_index.md [hunk 2]
```diff
@@ -41,7 +61,40 @@ Kubernetes divides workloads into different types. The most popular types suppor
...
+
+#### Multi-Cluster Application Upgrades
+
+When upgrading a multi-cluster application, you can configure an upgrade strategy to optimize network speed and upgrade reliability for your environment. 
+
+The batch size option lets you choose how many target-side workloads can upgraded simultaneously during rolling waves, optimizing network speed and upgrade reliability for your environment, just as you can with any other workload.
+
+## Workload Step-by-Step
+
+Follow the links below for step-by-step instructions on deploying workloads and using workload options.
```
Comment (alena1108) [inline content/rancher/v2.x/en/k8s-in-rancher/workloads/_index.md]: would this section change to reflect the multi cluster app deployment/upgrade and rollback?

File: content/rancher/v2.x/en/k8s-in-rancher/workloads/_index.md (hunk 1 above)
Comment (alena1108) [reply content/rancher/v2.x/en/k8s-in-rancher/workloads/_index.md]: Multi cluster app is always catalog templates based

File: content/rancher/v2.x/en/catalog/multi-cluster-apps/_index.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+![Multi-Cluster Application: Setup and Deployment]({{< baseurl >}}/img/rancher/multi-cluster-app.svg)
+
+
+Deploy multi-cluster applications using a catalog app. Following initial deployment, multi-cluster applications can be upgraded or rolled back just like any other workload.
+
+
+## Multi-Cluster Application Notes
+
+- When deploying a multi-cluster application, you can schedule it to either individual clusters/projects or _all_ of your clusters/projects.
```
Comment (alena1108) [inline content/rancher/v2.x/en/catalog/multi-cluster-apps/_index.md]: target "all" is not supported

File: content/rancher/v2.x/en/catalog/multi-cluster-apps/_index.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+- **Clone**
+
+    After creating a multi-cluster application, you can clone it and use it as a template for another multi-cluster application with a slightly different configuration. Cloning saves you the busy work of manually duplicating a configuration.
+
+- **Upgrade**
+
+    When a new version of an application is released, you can upgrade it across clusters by upgrading its multi-cluster application. When performing a multi-cluster upgrade, you'll have an opportunity to configure an upgrade strategy that optimizes network speed and upgrade reliability for your environment.
+
+    The **batch size** option lets you choose how many target-side workloads can be upgraded simultaneously during rolling waves, optimizing network speed and upgrade reliability for your environment, just as you can with any other workload.
```
Comment (ankur111) [inline content/rancher/v2.x/en/catalog/multi-cluster-apps/_index.md]: also include  "interval time"

**Comment by deniseschannon** [comment]
Closed in favor of https://github.com/rancher/docs/pull/1222

**Pull request closed without merging**